### ISSUE TITLE
3D StarDist for LC-OCT images

### ISSUE THREAD

**Issue by Optimox** [stardist/stardist#179]
Hi everyone!

I just wanted to share a personal good news : I recently published a paper in Nature Scientific Reports that uses 3D StarDist https://rdcu.be/cEGnl.

I wanted to share this with the developers of the repo! Congratulations for this nice library and thank you for sharing it publicly and maintaining it!

The paper shows how strong quantitative approaches can be based on 3D segmentation of cell nuclei and how Machine Learning techniques can quantify blurry notion as cellular atypia.

I am not sure this message belongs to the issue section, just wanted to share this with you, please feel free to close it!

Best,

Sébastien

**Comment by uschmidt83** [comment]
Congratulations on the paper and thanks for telling us!

**Comment by maweigert** [comment]
Indeed, congrats and thanks for letting us know, well appreciated! :)
